install-devnet-packages.sh

```diff
@@ -0,0 +1,530 @@
+        echo "jq not found, installing..."
+        brew install jq
+    else
+        echo "jq is already installed."
+    fi
+
+    if command -v pnpm &> /dev/null; then
+        pnpm install:foundry
+    else
+        echo "Pnpm is not installed. Skipping Foundry installation."
+    fi
+
+    echo
+
+    echo "All $TOTAL_STEPS steps are complete."
+
+# Linux specific steps
+elif [ "$OS_TYPE" = "Linux" ]; then
+
+    # If the operating system is Ubuntu, execute the following commands
+    if [ "$OS_NAME" = "Ubuntu" ]; then
+
+        if ! command -v sudo &> /dev/null; then
+            echo "sudo not found, installing..."
+            apt install -y sudo
+        else
+            echo "sudo is already installed."
+        fi
+
+        # 1. Update package list
+        echo "[1/$TOTAL_STEPS] ----- Updating package list..."
+        sudo apt-get update -y
+        sudo apt update -y
+
+        echo
+
+        # 2. Install Git
+        echo "[2/$TOTAL_STEPS] ----- Installing Git..."
+        if ! command -v git &> /dev/null; then
+            echo "git not found, installing..."
+            sudo apt install -y git
+        else
+            echo "git is already installed."
+        fi
+
+        echo
+
+        # 3. Install Make
+        echo "[3/$TOTAL_STEPS] ----- Installing Make..."
+        if ! command -v make &> /dev/null; then
+            echo "make not found, installing..."
+            sudo apt install -y make
+        else
+            echo "make is already installed."
+        fi
+
+        echo
+
+        # 4. Install Build-essential
+        echo "[4/$TOTAL_STEPS] ----- Installing Build-essential..."
+        if ! dpkg -s build-essential &> /dev/null; then
+            echo "Build-essential not found, installing..."
+            sudo apt install -y build-essential
+        else
+            echo "Build-essential is already installed."
+        fi
+
+        echo
+
+        # 5. Install Go (v1.22.6)
+        echo "[5/$TOTAL_STEPS] ----- Installing Go (v1.22.6)..."
+        if ! go version | grep "go1.22.6" &> /dev/null; then
+            echo "Go 1.22.6 not found, installing..."
+
+            if ! command -v curl &> /dev/null; then
+                echo "curl not found, installing..."
+                sudo apt install -y curl
+
+                if [ "$SHELL_NAME" = "zsh" ]; then
+                    source ~/.zshrc
+                elif [ "$SHELL_NAME" = "bash" ]; then
+                    source ~/.bashrc
+                    source ~/.profile
+                fi
+            else
+                echo "curl is already installed."
+            fi
+
+            sudo curl -L -o go1.22.6.linux-amd64.tar.gz https://go.dev/dl/go1.22.6.linux-amd64.tar.gz
+
+            sudo rm -rf /usr/local/go && tar -C /usr/local -xzf go1.22.6.linux-amd64.tar.gz
+
+            # Check if the Go configuration is already in the CONFIG_FILE
+            if ! grep -Fxq 'export PATH=$PATH:/usr/local/go/bin' "$CONFIG_FILE"; then
```

Comment:
double quotes is missed

`'export PATH="$PATH:/usr/local/go/bin"'`

Reply:
I applied it to commit [be172be](https://github.com/tokamak-network/tokamak-thanos/commit/be172be60cdcb4cfaea22c62dbb0daddcab267bd#diff-b5bd127c4255f4b6f64cc4d22ee510f24bc96d6f5353847bb1c8250f7356e6f7R328). Thank you.